src/components/ModelDisplayData.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+import { Info } from './Icons';
+
+import '../../css/model_display_data.css';
+import { ModelInfo } from '@aics/simularium-viewer/type-declarations/simularium/types';
+
+interface ModelDisplayDataProps extends ModelInfo {
+  trajectoryTitle?: string;
+}
+
+const ViewerTitle: React.FunctionComponent<ModelDisplayDataProps> = (
+  props: ModelDisplayDataProps
+): JSX.Element => {
+  const { title, trajectoryTitle } = props;
+
+  const hasMetaData = (): boolean => {
+    for (const key in props) {
+      if (key !== 'trajectoryTitle') {
+        if (props[key as keyof ModelDisplayDataProps] !== undefined) {
+          return true;
+        }
+      }
+    }
+    return false;
+  };
+
+  return (
+    <div className="title-container">
+      <div className="title">
+        {trajectoryTitle || title}
```

Comment:
I like Peytons idea of having a placeholder title. 
```suggestion
        {trajectoryTitle || title || "<Untitled trajectory>"}
```

Reply:
lol this commit broke the build somehow, i'll follow up shortly